frontend/src/components/TableUnratedTaskRow.tsx

```diff
@@ -113,60 +114,59 @@ export const TableUnratedTaskRow: React.FC<TableTaskRowProps> = ({ task }) => {
     }
   }, [task.ratings, allCustomers, allUsers, userInfo, type, task.roadmapId]);
 
-  const deleteTaskClicked = (e: React.MouseEvent<unknown, MouseEvent>) => {
+  const rateTaskClicked = (e: React.MouseEvent<any, MouseEvent>) => {
     e.preventDefault();
     e.stopPropagation();
-    dispatch(roadmapsActions.deleteTask({ id, roadmapId }));
+    dispatch(
+      modalsActions.showModal({
+        modalType: ModalTypes.RATE_TASK_MODAL,
+        modalProps: {
+          taskId: task.id,
+        },
+      }),
+    );
   };
 
-  const openModal = (modalType: ModalTypes) => (e: React.SyntheticEvent) => {
+  const taskDetailsClicked = (e: React.MouseEvent<any, MouseEvent>) => {
```

Comment:
Both taskDetailsClicked and rateTaskClicked could use openModal
```
const openModal = (modalType: ModalTypes) => (e: React.SyntheticEvent) => {
    e.preventDefault();
    e.stopPropagation();
    dispatch(
      modalsActions.showModal({
        modalType,
        modalProps: {
          taskId: task.id,
        },
      }),
    );
  };
```
The modal type would be sent on event handlers
`onClick={openModal(ModalTypes.TASK_INFO_MODAL)}`
`onClick={openModal(ModalTypes.RATE_TASK_MODAL)}`